simplq/src/components/pages/Admin/index.jsx

```diff
@@ -62,7 +68,7 @@ export default () => {
         queueId={queueId}
         className={styles.shareButton}
         onRefresh={() => {
-          update();
+          update(true);
           setItems(false);
```

Comment:
This seems redundant. Why was this added? Can it be removed safely?

Reply:
I believe it can

Reply:
Do you want to completely rewrite this page? not the HTML/Styling part, but the logic?

Reply:
Hmm ok. I'll take it up. And the queustatus file too